careers/technical-engineer.md

```diff
@@ -1,48 +1,78 @@
 
-## Staff Engineer
+Concrete examples could include:
 
-The Staff Engineer has strategic impact over some combination of a large team, a very deep technical problem, and/or a long time horizon. The problems that the staff engineer is solving are open-ended.
+- Lead or delegate extra-large projects across teams
+- Oversee cross-team initiatives (a11y, code quality, testing, reliability, etc.)
+- Solve problems that span multiple groups
 
-**Technical Ability:** The Staff Engineer provides considerable high-level technical guidance across the team. They can usually anticipate and plan for technical problems. They are highly knowledgeable in major parts of our technology stack and are the technical owner of significant components of our code base. Staff engineers have a sustained track record of creating improvements in business-critical systems around stability, performance, and scalability.
+**Engineering Perspective:** Staff-plus engineers are the folks who will often get unexpectedly pulled into the room where this sort of decision is happening. This gives them the opportunity to inject the engineering context and perspective into a decision while it's still possible to change the outcome. These brief moments of input on critical decisions are unduly impactful and will allow you to inject an engineering perspective where it would otherwise be missed.
 
-**Technical Scope:** A Staff Engineer is still acting in a very hands-on role, and as such, they are a prolific contributor to both core projects at Packback and to experimental work. When presented with a complex problem, process or existing system they are able to reduce the complexity in order to get more done with less work. They provide guidance, direction and help colleagues build and produce better outcomes.
+Concrete examples could include:
 
-**Communication:** The Staff Engineer has strong abilities to influence without requiring reporting authority. They facilitate cross-team work and are influential beyond their individual group. They are capable of driving groups of disparate interests to decisions, and clearly communicating and seeing those decisions through to impact. The staff engineer is a very strong communicator and can simplify technical concepts in a way to explain them to non-technical partners. The staff engineer is capable of setting short to medium term strategic direction for part of the technology stack, identifying areas of critical need based on future growth and developing roadmaps to attack those problems. The Staff Engineer may act as the leader of a technical team but is probably not acting as the people manager of fellow engineers.
+- Your perspective is sought out in critical decisions
```

Comment:
Having an example of "Engineering Perspective" be "your perspective was given" doesn't feel like much of an example. Is there maybe an example of a critical decision where a Staff Engineer's perspective would be desired / required?

The other three examples here are kind of in the same boat - this feels more like restating the paragraph above rather than concrete examples.